src/shadow-embed.js

```diff
@@ -61,7 +62,10 @@ export function createShadowRoot(hostElement) {
   // Native support.
   const shadowDomSupported = getShadowDomSupportedVersion();
   if (shadowDomSupported == ShadowDomVersion.V1) {
-    return hostElement.attachShadow({mode: 'open'});
+    // NOTE(torch2424) need to have an id on the shadow root no matter what
+    const shadowRoot = hostElement.attachShadow({mode: 'open'});
```

Comment:
Could you perhaps . give a bit more context on what you are referring to? 

the `createShadowRoot()` function creates the id for us, therefore I did not place it there. if you are referring to code cleanliness/duplication, then yes, i can definitely fix that. I just want to ensure I am clear.